OpenRA.Game/Renderer.cs

```diff
@@ -111,6 +111,7 @@ public void BeginFrame(int2 scroll, float zoom)
 
 		public void SetViewportParams(int2 scroll, float zoom)
 		{
+			// PERF: SetViewportParams is slow. Only call it when things change.
```

Comment:
It applies to the individual `*Render.SetViewportParams` calls, not the method itself